src/ucp/api/ucp.h

```diff
@@ -1344,6 +1351,20 @@ ucs_status_t ucp_mem_unmap(ucp_context_h context, ucp_mem_h memh);
 
 /**
  * @ingroup UCP_MEM
+ * @brief query mapped memory segment
+ *
+ * This routine returns valid address and length of memory segment mapped with
+ * @ref ucp_mem_map "ucp_mem_map()" routine.
+ *
+ * @param [in]  memh        @ref ucp_mem_h "Handle" to memory region.
+ * @param [out] params      Defined @ref ucp_mem_map_params_t configurations
+ *                          for the @ref ucp_mem_h "UCP memory handle".
+ */
+ucs_status_t ucp_mem_query(const ucp_mem_h memh, ucp_mem_map_params_t *params);
```

Comment:
the query function should return a new structure (not ucp_mem_map_params_t), according to the convention it would be `ucp_mem_attr_t`.